src/app/components/NavbarUSWDS.tsx

```diff
@@ -0,0 +1,72 @@
+"use client"
+import { Header, PrimaryNav } from "@trussworks/react-uswds";
+import Link from "next/link";
+import Image from "next/image";
+import { usePathname } from "next/navigation";
+import React from "react";
+import styles from "../styles/Home.module.scss";
+
+interface NavigationLinkProps {
```

Comment:
I think we could reuse a lot of what's in the `Footer` component and a lot of my same comments could apply here too, so I'll mostly skip this file for now 😄  I'll leave it up to you whether we'd like to make changes here now or do some clean up in a different PR.